src/runtime/rmswin.ri

```diff
@@ -1674,10 +1674,14 @@ HFONT mkfont(char *s, char is_3D)
        * This is a legal Icon font spec.
        * Check first for special "standard" family names.
        */
-      if (!strcmp(family, "mono") || !strcmp(family, "fixed")) {
-	 stdfam = "Lucida Sans"; /* Lucida Console? */
+     if (!strcmp(family, "mono")) {
```

Comment:
indentation is off

Reply:
That willl be the tabs (sigh).  OK, I'll fix it.  There are are times (most of the time, actually) when I wish we could replace **every** tab in the source code with spaces.